### DISCUSSION TITLE
what data types does C have?

### DISCUSSION THREAD

**Discussion by 11django11** [11django11/test-project#9] [Q&A] [answered]
type some examples pls

**Comment by seldish-og** [comment, marked as answer]
1 | Basic Types They are arithmetic types and are further classified into: (a) integer types and (b) floating-point types.
-- | --
2 | Enumerated types They are again arithmetic types and they are used to define variables  that can only assign certain discrete integer values throughout the  program.
3 | The type void The type specifier void indicates that no value is available.
4 | Derived types They include (a) Pointer types, (b) Array types, (c) Structure types, (d) Union types and (e) Function types.
